src/parts/Sidebar/SidebarContent/Navigation/index.tsx

```diff
@@ -68,6 +72,18 @@ const NAVIGATION_ITEMS = [
     icon: () => null,
     separator: true
   },
+  {
+    name: 'nav_crowdloan',
+    link: CROWDLOAN_DOMAIN,
+    icon: CashIcon,
+    external: true,
+    // This will suppress the link on testnet
+    suppressed: process.env.REACT_APP_BITCOIN_NETWORK !== 'mainnet',
```

Comment:
@nud3l @crypto-engineer is this the most reliable way to flag this? Requirement from Alexei was that we suppress the Claim Kint link on testnet.

Reply:
yeah, that's the best way to do it IMO.